### ISSUE TITLE
Compressed Flint

### ISSUE THREAD

**Issue by memyselfandcj** [AllTheMods/All-the-mods-7-Sky#69]
Not really an issue but not a mod suggestion either. With the amount of flint that you gather from sieving can we get a compressed flint block added for easier storage especially for early game??

Thanks!!

**Comment by KamiAso** [comment]
agree, or maybe this is a feature reserved for the implementation of Quark? Since Quark provides flint block and its an usual mod for ATM pack.
